src/core/ProgressBar/BaseProgressBar/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Theme } from '@material-ui/core';
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: '12.125rem',
+    borderRadius: '0.125rem',
+  },
+  label: {
+    fontSize: '0.75rem',
+    lineHeight: '0.875rem',
+    marginBottom: theme.spacing(1.625),
+    display: 'block',
```

Comment:
display block is present by default, we wouldn't require to explicitly mention it